tests/php/modules/contact-form/test-class.grunion-contact-form.php

```diff
@@ -867,28 +894,49 @@ public function assertValidFieldMultiField( $html, $attributes ) {
 
 			$this->assertEquals( $select->getAttribute( 'class' ), 'select '. $attributes['class'], ' select class does not match expected' );
 
-			// First Option
-			$option = $this->getFirstElement( $select, 'option' );
-			$this->assertEquals( $option->getAttribute( 'value' ), $attributes['values'][0], 'Input value doesn\'t match' );
-			$this->assertEquals( $option->getAttribute( 'selected' ), 'selected', 'Input is not selected' );
-			$this->assertEquals( $option->nodeValue, $attributes['options'][0], 'Input does not match the option' );
-
+			// Options.
+			$options = $select->getElementsByTagName( 'option' );
+			$n       = is_array( $options ) ? count( $options ) : $options->length;
+			$this->assertEquals( $n, count( $attributes['options'] ), 'Number of inputs doesn\'t match number of options' );
+			$this->assertEquals( $n, count( $attributes['values'] ), 'Number of inputs doesn\'t match number of values' );
+			for ( $i = 0; $i < $n; $i++ ) {
+				$option = is_array( $options ) ? $options[ $i ] : $options->item( $i );
```

Comment:
Same as above - will `$options` ever be an array?

Reply:
Same reply as above.